src/view/com/posts/FeedItem.tsx

```diff
@@ -196,6 +196,14 @@ let FeedItemInner = ({
     },
   ]
 
+  const isParentBlocked =
+    Boolean(
+      parentAuthor?.viewer?.blockedBy ||
+        parentAuthor?.viewer?.blocking ||
+        parentAuthor?.viewer?.blockingByList,
+    ) ||
+    (!parentAuthor?.displayName && !parentAuthor?.handle)
```

Comment:
This check feels a bit weird, what does the server give us?

Reply:
Good question, I should have looked deeper before. This was a hack for a 3p viewing a reply where the author and OP have a block between them. See latest commit, gonna ping in Slack.